### DISCUSSION TITLE
Changing timezone in docker container?

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#4898] [Q&A] [answered]
I wanna change the timezone to US/Eastern because items show as being published from a day ahead for me. I tried doing `docker exec <container ID> ln -sf /usr/share/zoneinfo/US/Eastern /etc/localtime` and `docker exec <container ID> tzselect` but it hasn't worked.

**Comment by Frenzie** [comment, marked as answer]
I'm not quite sure what you mean by didn't work. Do you mean timedatectl showed it as updated and it didn't work in FreshRSS or that it didn't do anything at all?

But regardless, see https://github.com/FreshRSS/FreshRSS/tree/edge/Docker. You can simply run the container with `-e TZ=US/Eastern` (or TZ: US/Eastern in Docker Compose).

**Reply by Alkarex** [reply]
See our Docker documentation for timezone (`TZ`): https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#quick-run

**Reply by ghost** [reply]
> I'm not quite sure what you mean by didn't work. Do you mean timedatectl showed it as updated and it didn't work in FreshRSS or that it didn't do anything at all?
> 
> But regardless, see https://github.com/FreshRSS/FreshRSS/tree/edge/Docker. You can simply run the container with `-e TZ=US/Eastern` (or TZ: US/Eastern in Docker Compose).

If i did `docker exec <container ID> date` it would still show up as the time and date from before, being ahead of me, the feeds also show being received from a time ahead of me. I already made the container, so I can't rerun the quick-run command because it says that container is already in use. I'm not sure I should delete alltogether and just restart the whole setup.

**Reply by Frenzie** [reply]
All relevant data is stored in the directory you passed to `freshrss_data`. You can remove the container whenever you like; it holds no value. The main reason to play with a container already running is to figure out what to put in the Dockerfile in the future. ^_^

Sidenote, which container did you use and did you check the exit status of your commands? For the Alpine one you'd need to `apk add --no-cache tzdata`. The Debian image includes it by default. @Alkarex we should probably add that to the Alpine image?

**Reply by Alkarex** [reply]
See patch https://github.com/FreshRSS/FreshRSS/pull/4903

**Reply by Alkarex** [reply]
See also new feature for user-defined time zone https://github.com/FreshRSS/FreshRSS/pull/4906
